README.md

```diff
@@ -144,7 +144,7 @@ More amazing notebooks here!
 | [221-machine-translation](notebooks/221-machine-translation)<br> | Real-time translation from English to German |  |
 | [222-vision-image-colorization](notebooks/222-vision-image-colorization/)<br> | Use pre-trained models to colorize black \& white images using OpenVINO | <img src = "https://user-images.githubusercontent.com/18904157/166343139-c6568e50-b856-4066-baef-5cdbd4e8bc18.png" width=225> |
 | [223-gpt2-text-prediction](notebooks/223-gpt2-text-prediction/)<br> | Use GPT-2 to perform text prediction on an input sequence | <img src=https://user-images.githubusercontent.com/91228207/185105225-0f996b0b-0a3b-4486-872d-364ac6fab68b.png  width=225> |
-
+| [224-stable-diffusion-text-to-image](notebooks/224-stable-diffusion-text-to-image)<br> | Use Stable Diffusion to perform text to image generation | <img src=notebooks/224-stable-diffusion-text-to-image/data/Corgi.png width=225>|
```

Comment:
I would rephrase it like "Text-to-image generation with Stable Diffusion method"

Reply:
rephrased